### ISSUE TITLE
Dimension Fail

### ISSUE THREAD

**Issue by TheEmpireSro** [YUNG-GANG/YUNGs-Better-End-Island#8]
Hey,

so I made the end portal with my pearls and when I got into the end portal everything seemed fine. but when I killed the crystals outside and tried to kill them in the middle in the house, they were undestroyable. I tried to reset by end_island rest. it did, but no dragon spawned and the crystals were in the middle...no outter ring no dragon.

**Comment by yungnickyoung** [comment]
You need to walk around the middle in creative or survival mode to activate the summoning ritual. The dragon does not spawn by default with this mod

**Comment by TheEmpireSro** [comment]
Look, the first time I moved into the end the dragon spawned imediatly. But I couldnt kill the 4 crystals in the house in the middle. The outter ring I could kill all. 

Then I did reset the end, relogged and moved again in the end. nothing happened. flew around almost 15 minutes. nothing. the outter crystals didnt spawn, the dragon didnt spawn, but the crystals in the house in the middle did spawn.

**Comment by yungnickyoung** [comment]
Was the mod added to an existing world? If so, make sure you're using the latest version of the mod and try using the `/end_island reset` command while you're in the End. That will attempt to revert the center island and dragon fight to the starting state. If that doesn't work, you can also try `/end_island reset true`, but only as a last resort. And definitely back up your world before running either command!

**Comment by TheEmpireSro** [comment]
No. I play the Allthemods 9 modpack. And the mods on their discord told me to write it in here. 

Okay I´ll try it. fank ju.
